2fa/v1/openapi.yaml

```diff
@@ -111,8 +126,9 @@ paths:
   '/2fa/v1/application/{applicationId}':
     get:
       tags:
-        - 2fa-application-service
-      summary: Get a 2fa application
+        - "Application Service"
+      summary: Get 2fa application
+      description: You can get a specific application by using the GET HTTP method and the application’s UUID
```

Comment:
I would write it as 

    Returns the specified application.

The GET is clear as well the format of the applicationId should be stated somehwere.

Reply:
It is too long. I will use something shorter.
The application ID should be described in the table below as you said in your next comment.